DirectProgramming/C++SYCL_FPGA/Tutorials/Features/ip_authoring_interfaces/mmhost/README.md

```diff
@@ -0,0 +1,372 @@
+(Code can be found under `part4_ddr_hosts`).
+```c++
+struct DDRIP {
+  using ParamsBl1 = decltype(sycl::ext::oneapi::experimental::properties{
+      sycl::ext::intel::experimental::buffer_location<kBL1>,
+      sycl::ext::intel::experimental::maxburst<8>,
+      sycl::ext::intel::experimental::dwidth<256>,
+      sycl::ext::oneapi::experimental::alignment<kAlignment>,
+      sycl::ext::intel::experimental::awidth<32>,
+      sycl::ext::intel::experimental::latency<0>});
+
+  using ParamsBl2 = decltype(sycl::ext::oneapi::experimental::properties{
+      sycl::ext::intel::experimental::buffer_location<kBL2>,
+      sycl::ext::intel::experimental::maxburst<8>,
+      sycl::ext::intel::experimental::dwidth<256>,
+      sycl::ext::oneapi::experimental::alignment<32>,
+      sycl::ext::intel::experimental::awidth<kAlignment>,
+      sycl::ext::intel::experimental::latency<0>});
+
+  sycl::ext::oneapi::experimental::annotated_arg<int *, ParamsBl1> x;
+  sycl::ext::oneapi::experimental::annotated_arg<int *, ParamsBl1> y;
+  sycl::ext::oneapi::experimental::annotated_arg<int *, ParamsBl2> z;
+  int size;
+
+  void operator()() const {
+#pragma unroll 4
+    for (int i = 0; i < size; ++i) {
+      z[i] = x[i] + y[i];
+    }
+  }
+};
+```
+
+## Building the `mmhost` Sample
+> **Note**: When working with the command-line interface (CLI), you should configure the oneAPI toolkits using environment variables.
+> Set up your CLI environment by sourcing the `setvars` script located in the root of your oneAPI installation every time you open a new terminal window.
+> This practice ensures that your compiler, libraries, and tools are ready for development.
+>
+> Linux*:
+> - For system wide installations: `. /opt/intel/oneapi/setvars.sh`
+> - For private installations: ` . ~/intel/oneapi/setvars.sh`
+> - For non-POSIX shells, like csh, use the following command: `bash -c 'source <install-dir>/setvars.sh ; exec csh'`
+>
+> Windows*:
+> - `C:\Program Files(x86)\Intel\oneAPI\setvars.bat`
+> - Windows PowerShell*, use the following command: `cmd.exe "/K" '"C:\Program Files (x86)\Intel\oneAPI\setvars.bat" && powershell'`
+>
+> For more information on configuring environment variables, see [Use the setvars Script with Linux* or macOS*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-linux-or-macos.html) or [Use the setvars Script with Windows*](https://www.intel.com/content/www/us/en/develop/documentation/oneapi-programming-guide/top/oneapi-development-environment-setup/use-the-setvars-script-with-windows.html).
+
+
+### On Linux*
+This design uses CMake to generate a build script for GNU/make.
+
+1. Change to the sample directory.
+
+2. Configure the build system for the Agilex® 7 device family, which is the default.
+
+   ```
+   mkdir build<N>
+   cd build<N>
+   cmake .. -DTYPE=PART<N>
+   ```
+
+   where `<N>` is: 
+   - `1` for `part1_pointers`
```

Comment:
I think that this is incorrect - it should be `PART1`, `PART2` etc. to match what's in your cmake file right?